server/sonar-server/src/main/java/org/sonar/server/computation/source/LastCommitVisitor.java

```diff
@@ -41,7 +41,6 @@
   private final BatchReportReader reportReader;
   private final MeasureRepository measureRepository;
   private final Metric lastCommitDateMetric;
-  private final Metric daysSinceLastCommitDateMetric;
   private final System2 system2;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "system2" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)